nltk/test/binary_fixt.py

```diff
@@ -0,0 +1,11 @@
+from nltk.internals import find_binary
+
+
+def check_binary(binary):
+    import pytest
+
+    try:
+        find_binary(binary)
+    except:
```

Comment:
Can we make this a bit more specific? I'd rather not keep it as just a catch-all.

Reply:
Ok, using LookupError now. OSError seems also possible, but it is not as specific.